cfme/tests/physical_infrastructure/api/test_physical_switch_api.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+import pytest
+from cfme.physical.provider.lenovo import LenovoProvider
+from cfme.utils.rest import assert_response
+
+pytestmark = [
+    pytest.mark.tier(3),
+    pytest.mark.provider([LenovoProvider], scope='module')
```

Comment:
Provider marker should be function scoped . 
Please change it to : 
pytest.mark.provider([LenovoProvider], scope='function')
 or 
pytest.mark.provider([LenovoProvider])